website/docs/plugins/ray_launcher.md

```diff
@@ -185,31 +198,99 @@ Loaded cached provider configuration
 16-32-25/2/checkpoint/checkpoint_3.pt
 ...
 [HYDRA] Stopping cluster now. (stop_cluster=true)
-[HYDRA] NOT deleting the cluster (provider.cache_stopped_nodes=true)
-[HYDRA] Running command: ['ray', 'down', '-y', '/var/folders/n_/9qzct77j68j6n9lh0lw3vjqcn96zxl/T/tmpy430k4xr.yaml']
+[HYDRA] Deleted the cluster (provider.cache_stopped_nodes=false)
+Destroying cluster. Confirm [y/N]: y [automatic, due to --yes]
+...
+No nodes remaining.
+
 ```
 </details>
 
 
 ##### Manage Cluster LifeCycle
-You can manage the Ray EC2 cluster lifecycle by configuring the two flags provided by the plugin:
+You can manage the Ray EC2 cluster lifecycle by configuring the flags provided by the plugin:
 
 - Default setting (no need to specify on commandline): Delete cluster after job finishes remotely:
 ```commandline
 hydra.launcher.stop_cluster=true
-hydra.launcher.ray.cluster.provider.cache_stopped_nodes=False
+hydra.launcher.ray.cluster.provider.cache_stopped_nodes=false
+hydra.launcher.teardown_cluster.workers_only=false
+hydra.launcher.teardown_cluster.keep_min_workers=false
```

Comment:
This is confusing - it's not super clear to me what is the difference between
```
hydra.launcher.ray.cluster.provider.cache_stopped_nodes
```
and 
```
hydra.launcher.teardown_cluster.workers_only
```

Could you add more details here?

Reply:
I see, it is confusing indeed. Here's what I'm seeing with those options:

| cached_stopped_nodes | workers_only | behavior |
|---------------------------------|--------------------|-------------|
| false | false | All nodes are terminated |
| false | true | Keeps head node running and terminates only worker node |
| true | false | Keeps both head node and worker node and stops both of them |
| true | true | Keeps both head node and worker node and stops only worker node |

Reply:
Thanks for trying every combo out! Let's add this to the doc for future reference.